saleor/plugins/openid_connect/utils.py

```diff
@@ -457,9 +457,13 @@ def _update_user_details(
         fields_to_save.append("email")
     if last_login:
         if not user.last_login or user.last_login.timestamp() < last_login:
-            login_time = make_aware(datetime.fromtimestamp(last_login))
+            login_time = timezone.make_aware(datetime.fromtimestamp(last_login))
             user.last_login = login_time
             fields_to_save.append("last_login")
+    else:
+        if not user.last_login or (timezone.now() - user.last_login).days > 1:
```

Comment:
Do not update last_login on each user request, but update it if it's older than a day